addons/tmf_loadouts/loadouts/uk_rm_1982.hpp

```diff
@@ -0,0 +1,512 @@
+
+	// These are put into the backpack
+	backpackItems[] = {"MNP_Beret_RM"};
+
+	// Unit traits, see https://community.bistudio.com/wiki/setUnitTrait
+	traits[] = {};
+
+	// This is executed after unit init is complete. argument: _this = _unit.
+	code = "";
+};
+
+class r : baseMan {
+	displayName = "Rifleman";
+	primaryWeapon[] = {"hlc_rifle_L1A1SLR"};
+	magazines[] = {
+		LIST_7("CUP_20Rnd_762x51_FNFAL_M"),
+		LIST_4("CUP_HandGrenade_L109A1_HE"),
+		"SmokeShell"
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	primaryWeapon[] = {"CUP_glaunch_M79"};
+	sidearmWeapon[] = {"CUP_hgun_Colt1911"};
+	magazines[] = {
+		LIST_20("CUP_1Rnd_HE_M203"),
+		LIST_5("CUP_1Rnd_StarFlare_Red_M203"),
+		LIST_5("CUP_1Rnd_Smoke_M203"),
+		LIST_5("CUP_1Rnd_SmokeRed_M203"),
+		LIST_10("CUP_7Rnd_45ACP_1911")
+	};
+};
+
+class car : r {
+	displayName = "Carabinier";
+	magazines[] = {
+		LIST_7("CUP_20Rnd_762x51_FNFAL_M"),
+		LIST_2("CUP_HandGrenade_L109A1_HE"),
+		"SmokeShell"
+	};
+};
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	traits[] = {"medic"};
+	backpack[] = {"CUP_B_ACRPara_dpm"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	backpack[] = {"usm_pack_alice"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : r {
+	displayName = "Submachinegunner";
+	primaryWeapon[] = {"hlc_smg_mp5a3"};
+	scope[] = {};
+	magazines[] = {
+		LIST_7("hlc_30Rnd_9x19_B_MP5"),
+		LIST_4("CUP_HandGrenade_L109A1_HE"),
+		"SmokeShell"
+	};
+};
+
+class ftl : smg {
+	displayName = "Fireteam Leader";
+	items[] += {"ACE_MapTools"};
+	sidearmWeapon[] = {"CUP_hgun_Colt1911"};
+	magazines[] += {
+		LIST_5("CUP_7Rnd_45ACP_1911")
+	};
+	linkedItems[] += {
+		"Binocular",
+		"ItemRadio"
+	};
+};
+
+class sl : ftl {
+	displayName = "Squad Leader";
+	backPack[] = {"usm_pack_st138_prc77"};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"MNP_Beret_RM"};
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	backpack[] = {"usm_pack_alice_prc77"};
+	backpackItems[] = { FAC_GEAR };
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	vest[] = {"usm_vest_LBE_mg"};
+	primaryWeapon[] = {"CUP_arifle_FNFAL5060"};
+	silencer[] = {"hlc_muzzle_mag58_brake"};
+	bipod[] = {"cup_bipod_fnfal"};
+	magazines[] = {
+		LIST_13("CUP_30Rnd_TE1_Red_Tracer_762x51_FNFAL_M"),
+		LIST_4("CUP_HandGrenade_L109A1_HE"),
+		"SmokeShell"
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpackItems[] = {
+		LIST_10("CUP_30Rnd_TE1_Red_Tracer_762x51_FNFAL_M")
+	};
+	linkedItems[] += {"Binocular"};
+};
+
+class rat : smg {
+	displayName = "Rifleman (AT)";
+	secondaryWeapon[] = {"CUP_launch_M72A6_Loaded"};
+	backpackItems[] = {
+		LIST_2("CUP_launch_M72A6_Loaded")
+	};
+};
+
+class dm : r {
+	displayName = "Designated Marksman";
+	primaryWeapon[] = {"hlc_rifle_L1A1SLR"};
+	scope[] = {"hlc_optic_suit"};
+};
+
+class mmgg : ar {
+	displayName = "MMG Gunner";
+	primaryWeapon[] = {"CUP_lmg_FNMAG"};
```

Comment:
L7

Reply:
The L7 doesn't have the wood furniture that was actually on the Brit version at the time. Can change if we want to, though.